tdms/include/globals.h

```diff
@@ -50,19 +50,21 @@ enum RCSType   { parallel , perpendicular };
 //			Mathematical Constants
 // **************************************
 
-const double dcpi  = 3.14159265358979323846;
-
+namespace tdms_math_constants
+{
+const double dcpi = 3.14159265358979323846;                                  // Pi
+const std::complex<double> IMAGINARY_UNIT = std::complex<double>(0.0, 1.0);  // Imaginary unit
+}
 
 // **************************************
 //			Physical Constants
 // **************************************
 
-const double eo	     =   8.85400e-12;				// free space electric permitivity
-const double mo	     =   4.0 * dcpi * 1.0e-7;		// free space magnetic permeability
-const double light_v =   1.0 / sqrt(mo*eo);			// free space light velocity
-const double Zo		 = 376.734;						// free space inpedance
-
-const double zero			   = 1.0e-12;			// zero level for numerical comparisons
-const double convergence_level = 1.0e-10;			// zero level for convergence
+namespace tdms_phys_constants
+{
+const double eo = 8.85400e-12;                               // free space electric permitivity
+const double mo = 4.0 * tdms_math_constants::dcpi * 1.0e-7;  // free space magnetic permeability
+const double light_v = 1.0 / sqrt(mo * eo);                  // free space light velocity
+const double Zo = 376.734;                                   // free space inpedance

```

Comment:
As you say, the notmal math convention has these guys as: $\epsilon_0$, $\mu_0$, $c$, and $Z_0$.

Well `0`!= `o` (is `o` chosen because it looks like a sub?). And (a matter of taste, but) would `kEpsilon0`, `kMu0` (or `EPSILON0`, `MU0`) be more readable in your opinion?

Again: if we're doing noisy changes anyway, let's get it right first time.

Reply:
Agreed, I thought it odd when I did my first pass but otherwise accepted it. With going over to SciComp standards, I'll replace these with `0`s.